File: python/leetcode009.py
class Solution:
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0:
            return False
        num = x
        pal = 0
        while num > 0:
            pal = pal*10 + num%10
            num = num//10
        if pal == x:
            return True
        else:
            return False

File: leetcodec++/leetcode009.cpp
//
// Created by 黄敏辉 on 2019-04-25.
//


class Solution {
public:
    bool isPalindrome(int x) {
        if (x < 0)
            return false;
        long int rev = 0;
        int tmp = x;
        while (tmp > 0){
            rev = rev * 10 + tmp % 10;
            tmp /= 10;
        }
        return rev == x;
    }
};